extensions/interactions/ItemSelectionInput/directives/oppia-interactive-item-selection-input.component.ts

```diff
@@ -87,6 +99,27 @@ export class InteractiveItemSelectionInputComponent implements OnInit {
       this.userSelections[this.choices[i]] = false;
     }
 
+    // Setup voiceover.
+    this.displayedCard = this.playerTranscriptService.getCard(
+      this.playerPositionService.getDisplayedCardIndex());
+    this.recordedVoiceovers = this.displayedCard.getRecordedVoiceovers();
+
+    // Combine labels for voiceover.
+    let combinedChoceLabels = '';
+    for (const choiceLabel of this.choices) {
+      // If the labels are in html format, remove the tags and leave the
+      // content only.
+      const cleanChoiceLabel = choiceLabel.replace(/<[^>]+>/g, '');
```

Comment:
Wouldn't this break if the content contains `<`?

Reply:
Not really: if it has a tag yes, but not if it has just a `<`.

Please see the following proof:
<img width="604" alt="Screen Shot 2022-04-28 at 11 25 19 PM" src="https://user-images.githubusercontent.com/2841740/165894463-fc22c049-bee3-4805-9ab5-cc5fcc5596c2.png">